### DISCUSSION TITLE
Wiki examples improvements

### DISCUSSION THREAD

**Discussion by tsymbaliuk** [COVESA/vsomeip#530] [Q&A]
Could you please update the Wiki in the Subscribe example? There should be
```
app->request_event(SAMPLE_SERVICE_ID, SAMPLE_INSTANCE_ID, SAMPLE_EVENT_ID, its_groups, vsomeip::event_type_e::ET_FIELD);
```
instead of
```
app->request_event(SAMPLE_SERVICE_ID, SAMPLE_INSTANCE_ID, SAMPLE_EVENT_ID, its_groups, true);
```
While here, consider defining `SAMPLE_EVENTGROUP_ID` and `SAMPLE_EVENT_ID` somewhere in the example code.

**Comment by nachtfuchs** [comment]
I also think that 
`app->request_event(SAMPLE_SERVICE_ID, SAMPLE_INSTANCE_ID, SAMPLE_EVENT_ID, its_groups, vsomeip::event_type_e::ET_FIELD);` needs an additional argument of type "reliability_type_e".
